framework_tvb/tvb/interfaces/rest/client/main_api.py

```diff
@@ -27,19 +27,35 @@
 #   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
 #
 #
-from tvb.interfaces.rest.commons import Strings
+import requests
+from tvb.interfaces.rest.commons.strings import Strings
 
 
 class MainApi:
     """
     Base API class which will be inherited by all the API specific subclasses
     """
 
-    def __init__(self, server_url):
+    def __init__(self, server_url, auth_token=''):
+        # type: (str, str) -> None
         """
         Rest server url, where all rest calls will be made
+        :param server_url: REST server base URL
+        :param auth_token: Keycloak authorization token
         """
         self.server_url = server_url + "/" + Strings.BASE_PATH.value
+        self.authorization_token = auth_token
```

Comment:
What is happening with long running operation?
Where can we check /configure the token attributes (expiration date) ?

Reply:
From keycloak server we receive also the number of seconds left before the access token expires.

Reply:
then it is configured in Keycloak, right?
What is the current/default ?

Reply:
Yes, it is configured in keycloak. Default value is 5 minutes